### PR TITLE
a few improvements to pybullet motion planning

### PR DESCRIPTION
**Pull request by tomsilver**
and a new test that finds motion plans like this:

https://user-images.githubusercontent.com/2816502/219951611-848b3b7f-1f61-4382-826c-713df790879b.mov

the new test takes about 0.7 seconds.

### CODE REVIEW

File: tests/pybullet_helpers/test_motion_planning.py [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+from predicators.pybullet_helpers.joint import JointPositions
+from predicators.pybullet_helpers.link import get_link_state
+from predicators.pybullet_helpers.motion_planning import run_motion_planning
+from predicators.pybullet_helpers.robots import \
+    create_single_arm_pybullet_robot
+
+USE_GUI = False
+
+
+def test_run_motion_planning(physics_client_id):
```
Comment (tomsilver) [inline tests/pybullet_helpers/test_motion_planning.py:20]: this test is just moved from `test_pybullets_robot.py`

File: tests/pybullet_helpers/test_motion_planning.py [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+                               joint_initial,
+                               joint_target,
+                               collision_bodies={table_id, block_id},
+                               seed=seed,
+                               physics_client_id=physics_client_id)
+    assert path is None
+    p.removeBody(block_id, physicsClientId=physics_client_id)
+
+
+def test_move_to_shelf():
```
Comment (tomsilver) [inline tests/pybullet_helpers/test_motion_planning.py:134]: this is the new test

File: tests/pybullet_helpers/test_motion_planning.py [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+                                   physicsClientId=physics_client_id)
+        p.configureDebugVisualizer(p.COV_ENABLE_RGB_BUFFER_PREVIEW,
+                                   False,
+                                   physicsClientId=physics_client_id)
+        p.configureDebugVisualizer(p.COV_ENABLE_DEPTH_BUFFER_PREVIEW,
+                                   False,
+                                   physicsClientId=physics_client_id)
+        p.configureDebugVisualizer(p.COV_ENABLE_SEGMENTATION_MARK_PREVIEW,
+                                   False,
+                                   physicsClientId=physics_client_id)
```
Comment (williamshen-nz) [inline tests/pybullet_helpers/test_motion_planning.py:199]: Might be worth moving these to some utils as it's used a few times over different modules: https://github.com/Learning-and-Intelligent-Systems/predicators/search?q=p.configureDebugVisualizer

**Review by williamshen-nz — APPROVED** [review]
LGTM :shipit:

**Pull request merged**